JS-Algorithms/challenges.js

```diff
@@ -95,7 +120,20 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+  let factorial = 1n;
```

Comment:
I went for the 'n' approach to make the code a bit more minimalistic,  but now that I noticed, I could've used "i = 1n" or "i=BigInt(1)" in my for loop instead of directly in the pow operation.

Reply:
I would say *n values are not commonly used but yes please use JS values in a better way